src/core/config/border_agent.h

```diff
@@ -122,6 +122,19 @@
 #define OPENTHREAD_CONFIG_BORDER_AGENT_MESHCOP_SERVICE_BASE_NAME "OpenThread BR (unspecified vendor) "
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_BORDER_AGENT_TRACKER_ENABLE
+ *
+ * Define to 1 to enable the Border Agent Tracker module.
```

Comment:
Should module --> feature?  Since that's used everywhere else

Reply:
Thanks. Updated in new pushed commit.